### PR TITLE
Adding Javascript

### PR DESCRIPTION
**Pull request by kautuk98**

### CODE REVIEW

File: templateLiterals.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+//This is the ES6 syntax
+function sides(literals, ...expressions) {
+  let [a, p] = expressions;
+
+  let s1 = (p - Math.sqrt(p * p - 16 * a)) / 4;
+  let s2 = (p + Math.sqrt(p * p - 16 * a)) / 4;
+
+  return [s1, s2].sort();
```
Comment (akshay-zzZ) [inline templateLiterals.js:8]: nice code

**Comment by kautuk98** [comment]
Heyy

**Comment by akshay-zzZ** [comment]
heya

File: templateLiterals.js (hunk 1 above)
Comment (kautuk98) [reply templateLiterals.js:8]: Thank you bro

Comment (akshay-zzZ) [reply templateLiterals.js:8]: welcome bro

**Review by akshay-zzZ — APPROVED** [review]

**Pull request merged**